<launch>
  <node pkg="airsim_img_publisher"
    type="airsim_imgPublisher"
    name="airsim_imgPublisher"
    output="screen">

    <!-- IP of the Airsim computer -->
    <param name="Airsim_ip" value="localhost"/>

    <!-- Port of the Airsim computer (this is default, do not change!) -->
    <param name="Airsim_port" value="42451"/>

    <!-- RGB camera parameters (parameters when calibrated) -->
    <param name="Fx" value="320"/>
    <param name="Fy" value="320"/>
    <param name="cx" value="320"/>
    <param name="cy" value="240"/>
    <param name="width" value="640"/>  <!-- Width when calibrated!-->
    <param name="height" value="480"/> <!-- Height when calibrated!-->

    <param name="localization_method" value="ground_truth" />

    <!-- Depth baseline -->
    <param name="Tx" value="0.14"/> 
    <param name="scale" value="0.38"/> 

    <!-- Topic with odometry information for the vehicle (used to create tf tree) -->
    <param name="odomTopic" value="/mavros/local_position/odom"/>


  </node>
</launch>

